rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -413,3 +413,40 @@ def test_import_snapshot(rotkehlchen_api_server, tmpdir_factory):
         contained_in_msg='csv file has invalid headers',
         status_code=HTTPStatus.CONFLICT,
     )
+
+
+def test_delete_snapshot(rotkehlchen_api_server):
+    conn = rotkehlchen_api_server.rest_api.rotkehlchen.data.db.conn
+    ts = Timestamp(ts_now())
+    _populate_db_with_balances(conn, ts)
+    _populate_db_with_location_data(conn, ts)
+    rotkehlchen_api_server.rest_api.rotkehlchen.data.db.set_settings(ModifiableDBSettings(main_currency=A_EUR))  # noqa: E501
+    response = requests.delete(
+        api_url_for(
+            rotkehlchen_api_server,
+            'dbsnapshotdeletingresource',
+        ),
+        json={
+            'timestamp': ts,
+        },
```

Comment:
nitpick: Don't use so many lines if you don't need to.

```suggestion
        json={'timestamp': ts},
```